src/application/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { DomainCustomException } from 'src/domain/common/errors/domain-custom-exception';
+import { DomainErrorCode } from 'src/domain/common/errors/domain-error-code';
+import { ProfileService } from 'src/domain/profile/profile.service';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy, 'jwt') {
+  constructor(private readonly profileService: ProfileService) {
+    const secret = process.env.JWT_ACCESS_SECRET;
+    if (!secret) {
+      throw new Error('JWT_ACCESS_SECRET 환경변수가 설정되어 있지 않습니다.');
```

Comment:
```suggestion
      throw new InternalServerError('JWT_ACCESS_SECRET 환경변수가 설정되어 있지 않습니다.');
```
환경 변수 내용을 못 읽어 온 것이니, 서버 에러에 더 가깝지 않을까요?

Reply:
환경변수가 설정되어있지 않으면, 서버를 실행시켜도 저 코드로 에러를 던지고 종료됩니다.

그래서 응답을 줄 필요가 없다고 생각했습니다.